integration/docker_engine_test.go

```diff
@@ -0,0 +1,73 @@
+// +build integration
+
+package integration
+
+import (
+	"context"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/aquasecurity/trivy/internal"
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRun_WithDockerEngine(t *testing.T) {
+	testCases := []struct {
+		name               string
+		testfile           string
+		expectedOutputFile string
+		expectedError      error
+	}{
+		{
+			name:               "happy path, alpine:3.10",
```

Comment:
I feel it is better to add a simple failing test. Even if this test always passes with any input, we can't know it without failing tests.

Reply:
Do you have a failing test in mind? I can't think of how we should make this fail.

Reply:
IMHO, we don't have to think of a difference of API version here. We can add a test to specify the wrong image name when scanning by Trivy.